HISTORY.rst

```diff
@@ -14,6 +14,8 @@ History
   * ``dlocal``
   * ``onpay``
   * ``safecharge``
+* Adds ``rule_label`` to minFraud output ``/disposition``.
+* Adds ``was_3d_secure_successful`` to ``/credit_card`` validation
```

Comment:
The tense on these differs from the other entries.

Reply:
Thanks!